src/bootstrap/native.rs

```diff
@@ -155,6 +155,7 @@ impl Step for Llvm {
            .define("WITH_POLLY", "OFF")
            .define("LLVM_ENABLE_TERMINFO", "OFF")
            .define("LLVM_ENABLE_LIBEDIT", "OFF")
+           .define("LLVM_OCAML_INSTALL_PATH", "usr/lib/ocaml")
```

Comment:
If the point of this environment is to declare an invalid binary so LLVM can't find the real `ocaml`, could we make it more explicit like

```rust
           .define("LLVM_OCAML_INSTALL_PATH", "/__disable_ocaml_bindings")
```

The current value looks like a typo 😄

Reply:
No, it's so it installs ocaml bindings in there instead of /usr/lib/ocaml

Reply:
basically ocaml will be found automagically, and the bindings will be installed in LLVM_OCAML_INSTALL_PATH

Reply:
@roblabla  Ah ok. Could you copy this explanation into the code?